ui-speedspacechart/src/components/layers/GridX.tsx

```diff
@@ -4,13 +4,13 @@ import type { Store } from '../../types/chartTypes';
 import { drawGridX } from '../helpers/drawElements/gridX';
 import { useCanvas } from '../hooks';
 
-type AxisLayerXProps = {
+type GridXProps = {
   width: number;
   height: number;
   store: Store;
 };
 
-const AxisLayerX = ({ width, height, store }: AxisLayerXProps) => {
+const GridX = ({ width, height, store }: GridXProps) => {
```

Comment:
It doesn't contains the ticks. It feels weird to named it axis since it's more a layer over the x axis.

Reply:
I still have a problem with that:
- every layers have the word `layer` in their name, except `GridX.tsx`
- `TickLayerY.tsx` contains `drawAxisY` that draws ticks, text, minor and major lines.
- `TickLayerX.tsx` contains `drawTickX` that draws only ticks and texts.
- `GridX.tsx` contains `drawGridX` that draws lines, steps and texts.

we're losing clarity, we should keep consistent names, maybe folders to keep an optimal readability and navigation

Reply:
I suggest: `StepsLayer`.

- It's closer to what this layer represents
- It contains the word `Layer`

Reply:
I named it steps because even if this is obvious for us, I'm no sure this is really relevant for anyone outside of the project how could use this module